src/stores/AccountController.ts

```diff
@@ -389,6 +391,34 @@ export async function reloadActiveSubscription({ delay }: { delay: number } = {
   });
 }
 
+/**
+ * Get multiple media items for the given IDs. This performs multiple GET requests and optionally fetches the tokens
+ * needed when URL signing (with DRM) is enabled.
+ *
+ * @todo This function is temporarily until we add support for the watchlist playlist
+ * {@link https://github.com/jwplayer/ott-web-app/issues/43}. We could also refactor the favorites and watch history
+ * playlist to use the `usePlaylist` hook as well.
+ *
+ * @param mediaIds
+ */
+export async function getMediaItems(mediaIds: string[]) {
+  const jwt = useAccountStore.getState().auth?.jwt;
+  const signingConfig = useConfigStore.getState().config.contentSigningService;
+  const host = signingConfig?.host;
+  const drmPolicyId = signingConfig?.drmPolicyId;
+  const drmEnabled = !!drmPolicyId && !!signingConfig?.drmEnabled;
+  const signingEnabled = !!host && drmEnabled && !!drmPolicyId;
```

Comment:
These property checks are done in 3 different places and they are all a little different (`AccountController`, `useSignedUrl`, and `usePlaylistItemCallback`). Can we just make host required on `contentSigningService` (but the entire block still optional) and drop the `drmEnabled`, so we just assume drm is enabled if `drmPolicyId` is populated?

Reply:
Yes, sounds good. I've consciously kept the `drmEnabled` as config so this can be controlled by the property settings later. But for now, it will make it more simple.

There are differences between the metadata and video calls. E.g. all media/playlist calls only need signing when DRM is enabled in the property whilst the media item call for video playback needs to be signed for URL signing and DRM.